src/anomalib/deploy/inferencers/torch_inferencer.py

```diff
@@ -82,9 +82,16 @@ def load_model(self, path: str | Path) -> nn.Module:
             (AnomalyModule): PyTorch Lightning model.
         """
 
-        model = torch.load(path, map_location=self.device)["model"]
-        model.eval()
-        return model.to(self.device)
+        try:
+            model = torch.load(path, map_location=self.device)["model"]
+            model.eval()
+            return model.to(self.device)
+        except Exception as e:
+            raise Exception(
```

Comment:
As suggested by Codacy, should we raise (and maybe catch) a more specific exception here?